build.gradle

```diff
@@ -221,6 +221,9 @@ repositories {
 ext["jackson.version"] = fasterxml_version
 ext["junit-jupiter.version"] = junit_version
 
+ext { qDox = "com.thoughtworks.qdox:qdox:2.1.0" }
```

Comment:
can you use a longer name here?

Reply:
Sure. I changed the name to `qDoxVersionReuseable.`